website/src/components/Messages/RenderedMarkdown.tsx

```diff
@@ -70,6 +70,9 @@ const sx: SystemStyleObject = {
       borderBottomColor: "gray.700",
     },
   },
+  "ol li::marker": {
+    content: "counter(list-item) '.'",
+  },
```

Comment:
could you please explain this fix to me?

Reply:
Whoop sorry, I didn't see your comment. Here is more information about this https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Counter_Styles/Using_CSS_counters